platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/stats/ReconnectMapMetrics.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.merkle.synchronization.stats;
+
+import com.swirlds.metrics.api.Counter;
+import com.swirlds.metrics.api.Metrics;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Objects;
+
+/**
+ * An implementation of ReconnectMapStats that emits all the stats as Counter metrics.
+ */
+public class ReconnectMapMetrics implements ReconnectMapStats {
+
+    private static final String RECONNECT_MAP_CATEGORY = "ReconnectMap";
```

Comment:
Current VM / DS metric categories are `virtual_map` and `merkle_db`, correspondingly. Should this category be `reconnect`, or `reconnect_vmap`, or something alike, then?

Reply:
Sounds good. I changed it to `reconnect_vmap`.